radixdlt/src/integration/java/com/radixdlt/consensus/deterministic/DeterministicTest.java

```diff
@@ -145,10 +163,25 @@ public void processNextMsg(int toIndex, int fromIndex, Class<?> expectedClass) {
 	}
 
 	public void processNextMsg(Random random) {
-		processNextMsg(random, (c, m) -> true);
+		processNextMsgWithReceiver(random, (c, m) -> true);
 	}
 
-	public void processNextMsg(Random random, BiPredicate<Integer, Object> filter) {
+	public void processNextMsg(Random random, Predicate<Object> filter) {
```

Comment:
Maybe document this method to describe the behaviour of the `filter`? That returning `true` results in messages being "sent" to `nodes`?

Reply:
Also not part of PR.